pennylane/ops/qubit/matrix_ops.py

```diff
@@ -67,18 +66,6 @@ def __init__(self, *params, wires, do_queue=True):
                     f"Input unitary must be of shape {(dim, dim)} to act on {len(wires)} wires."
                 )
 
-            # Check for unitarity; due to variable precision across the different ML frameworks,
-            # here we issue a warning to check the operation, instead of raising an error outright.
-            if not qml.math.allclose(
```

Comment:
are there any consequences to removing this?

Reply:
The solution in #2249 is much nicer than this, it will only make the unitarity check if the input is not abstract. I will wait until that is merged, then pull the changes into here and revert some of the test that I had originally updated to not perform the check.